starknet_py/net/models/transaction.py

```diff
@@ -171,60 +172,29 @@ class Invoke(AccountTransaction):
     calldata: List[int] = field(
         metadata={"marshmallow_field": fields.List(fields.String())}
     )
-    nonce: Optional[int] = field(metadata={"marshmallow_field": NoneFelt()})
```

Comment:
I have to add this comment here :/

Reply:
DeployAccount has a `version` field which is already present in the Transaction class, so is unnecessary here